packages/@dcl/inspector/src/lib/data-layer/host/rpc-methods.ts

```diff
@@ -13,8 +13,11 @@ export function initRpcMethods(fs: Fs, engine: IEngine): DataLayerInterface {
     // This method receives an incoming message iterator
     // and returns an async iterable. consumption and production of messages
     // are decoupled operations
-    stream: function (iter) {
-      return stream(iter, { engine })
+    async *stream(iter, ctx) {
+      const gen = stream(iter, { engine })
+      for await (const it of gen) {
```

Comment:
whats the logic behind this ? 
I think stream fn already does this.

Reply:
just something wrong in the types, adding a TODO